modxcms/revolution
# Fix trash page permission issues

### What does it do?
This fixes the issues with the permission checks related to the trash manager. The context menu was missing for any non-sudo users. Also added the menu_trash permission to the Administrator access policy by default.

![Screenshot 2021-03-05 at 15 35 42](https://user-images.githubusercontent.com/7802465/110129680-7e403380-7dc8-11eb-9fd7-f3242ef96029.png)

### Why is it needed?
By default a non-sudo user would not have the context menu on the resource trash manager page. Also the Administrator access policy was lacking the menu_trash permission by default, so non sudo administrator users would not be able to visit the trash manager.

### How to test

1. Create a new user (non sudo)
2. Assign Administrator Policy access policy
4. Delete a resource
5. Switch to trash manager, you should be able to access this page now
6. Try right click on a deleted resource, the context menu should now be shown


### Related issue(s)/PR(s)
https://github.com/modxcms/revolution/issues/14337
